request.go

```diff
@@ -134,16 +134,6 @@ func (r *Requester) SetClient(client *http.Client) *Requester {
 	return r
 }
 
-func (r *Requester) parseQueryString(queries map[string]string) string {
```

Comment:
Here's another issue that talks about the same kind of thing:  alecthomas/gometalinter#98